addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -69,10 +66,12 @@ html[data-edit_translations="1"] {
     }
     &.d-md-none::before, &.d-lg-none::before {
         height: 50px;
+        width: 50px;
         -webkit-mask: url("/website/static/src/img/snippets_options/desktop_invisible.svg") no-repeat 100% 100%;
     }
     &:not(.d-md-none):not(.d-lg-none)::before {
         height: 30px;
+        width: 30px;
         -webkit-mask: url("/website/static/src/img/snippets_options/mobile_invisible.svg") no-repeat 100% 100%;
```

Comment:
Add `pointer-events: none;` to this pseudo-element as well. For example, in this screenshot, we should be able to select the image of the column even when clicking on the "desktop invisible" icon. In fact, it might be the only thing to do in this PR, there's no need to change the width rules.
![image](https://github.com/user-attachments/assets/92299f60-e714-49cd-8bb5-864d1986ef1f)

Reply:
Hi @bvr-odoo Thanks for the review! I changed solution with `pointer-events : none;`
now clicking on icon itself does not interfere interaction with underlying element which was shortcoming of previous solution.